### DISCUSSION TITLE
Use Pcsx_rearmed standalone as the default PSX emulator

### DISCUSSION THREAD

**Discussion by iconoclusterdotexe** [OnionUI/Onion#591] [General]
It's much better than the RA core version: no geometry glitches (see Crash 3 Warped intro background), more PS1 games compatibility and the "enhanced resolution" options works much better than it's core counterpart.

**Comment by Olywa** [comment]
Some more testing and work is needed I believe. Default emulators should ideally be compatible with Onions Game Switcher functionality (currently this one is not) and while many games appear to perform better in the standalone, there will be some more resource heavy games that may run better in the current default core. 

It's certainly a consideration for the future, there has been a lot of discussion around it and @schmurtzm did a fantastic job working on the standalone and bringing it into the release.

**Comment by schmurtzm** [comment]
Hi,
The standalone works a little better but including it in Onion with a good user experience require a lot of work so I think it will stay experimental 😉
You can try to enable frameskip (may be it requires to switch audio output to sdl instead of oss) and enhanced resolution on the default core, the result is generally not so bad.

About the compatibility it should be similar because the core is updated at the same time than the standalone , are you sure that your onion Retroarch is up to date ?
You can reinstall Onion to try…

ps: I’m interested by games names with compatibility differences

**Reply by iconoclusterdotexe** [reply]
I've tested with 4.0.3 and 4.0.3dev versions. Same results.
You can test games like Duke Nukem: Time to Kill for the slowdown issue in core version (the standalone one can run it smoothly with enhanced resolution too) and Crash 3 Warped for geometrical background glitches in core version too (the developer/publisher logos screen at start with the purple/blue space background)

**Reply by schmurtzm** [reply]
Ok, but you don't have games which run on standalone and not on core version ? (just to know if we have something to dig on this point)

Otherwise yes, the standalone is nice but we can't put it as default for everyone because to implement save states, game switcher, etc it would require a lot of modifications in the standalone code to receive a signal from the Game Switcher. Futhermore I have some black screen on exit , not you ? (I suspect the included library on this point).

But in all case you cn easily put it as default for your own use.

**Reply by iconoclusterdotexe** [reply]
I don't have found any not running games in all of two pcsx_rearmed version, just many differences with some game or some perfomance drops with the core version. I've tested only 15-20 games in total with all of two versions. If i found some not working games i'll tell you faster ;)

For the Game Switcher feature I'm with you for using core version as default. I don't have tried that with the standalone version, but i can test it too.
